common/theme.c

```diff
@@ -95,36 +95,43 @@ void load_theme(struct theme_config *theme, struct mux_config *config, struct mu
 
     theme->NAV.A.GLYPH = get_ini_hex(muos_theme, "navigation", "NAV_A_GLYPH");
     theme->NAV.A.GLYPH_ALPHA = get_ini_int(muos_theme, "navigation", "NAV_A_GLYPH_ALPHA", 255);
```

Comment:
Added recolour alpha settings so that by default existing themes keep their original colors for footer glyphs